androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivity.java

```diff
@@ -154,6 +154,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
                         .setColorSchemeParams(
                                 CustomTabsIntent.COLOR_SCHEME_DARK, darkModeColorScheme)
                         .setDisplayMode(getDisplayMode());
+                       .setScreenOrientation(mMetadata.screenOrientation);
```

Comment:
This method doesn't exist yet, right?

Also, nit: indentation

Reply:
`mMetadata.screenOrientation` will be null here, if the property is not defined in `AndroidManifest.xml`.

Reply:
@PEConn The setScreenOrientation is not existed yet, we need to add that for the twaIntentBuilder class in androidX.